src/test/java/com/qunite/api/web/QueueControllerIntegrationTest.java

```diff
@@ -67,6 +65,7 @@ public void cleanAll() {
   }
 
   @Test
+  @WithMockUser("First")
```

Comment:
Apply this annotation to class and not to each test

Reply:
Due to changes in structure of applications and so in tests there probably will be situations where we will need different users for different tests, so i don`t think that binding all tests to a certain user is a good idea.